introduction/hardware-options/additional-peripheral-devices/relays.md

```diff
@@ -0,0 +1,10 @@
+# Relays
+
+Relays are useful for turning higher voltage (>5V DC) auxiliary equipment on and off. 
+
+Up to three relays can be operated via joystick button functions when connected to the appropriate signal outputs.
```

Comment:
"that can be configured via [QGC](operators-manual/button-functions.html)"

Reply:
Link added. This will need to be changed when the new joystick button section is merged.